Tests/UnitTests/FoundationExtensions/DecoderExtensionTests.swift

```diff
@@ -260,14 +260,36 @@ class DecoderExtensionsDefaultDecodableTests: TestCase {
     }
 
     func testDecodesEmptyArrayForIncorrectType() throws {
-        let json = "{\"list\": \"this is not an array\"}"
+        let json = "{\"array\": \"this is not an array\"}"
```

Comment:
These tests were wrong cause this was the wrong key, so of course the test failed, since there was no data.